src/pretix/static/pretixcontrol/scss/_forms.scss

```diff
@@ -588,6 +588,9 @@ table td > .checkbox input[type="checkbox"] {
     border-bottom: 1px solid #ccc;
   }
 }
+.form-horizontal .big-radio-wrapper .radio label > span {
+  display: block;
+}
```

Comment:
To match the existing big-radio in product settings, we need to make the radio-labels bold and add a little space at the top of the help-text. Furthermore I added more spacing to the top of the control-label to match the baseline of the first radio-label (this is an improvement to the big-radio in product-settings as well).

```suggestion
.form-horizontal .control-label:has(+.big-radio-wrapper),
.form-horizontal .control-label:has(+div > .big-radio) {
  padding-top: 16px;
}
.form-horizontal .big-radio-wrapper .radio label {
  font-weight: bold;
}
.form-horizontal .big-radio-wrapper .radio label > span {
  display: block;
  font-weight: normal;
}
```
![Bildschirmfoto 2024-10-11 um 10 05 50](https://github.com/user-attachments/assets/373684c3-f367-42d3-ad17-24200022a1fd)
![Bildschirmfoto 2024-10-11 um 10 06 16](https://github.com/user-attachments/assets/1e341e49-a765-4800-b35a-35c60c3031ee)